plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -101,7 +103,18 @@ class Bazel(
         // resolution.
         val registryUrl = parseLockfile(lockfile).registryUrl()
 
-        val packages = getPackages(scopes, registryUrl)
+        val registry = if (registryUrl.orEmpty().startsWith("file://")) {
```

Comment:
Better leverage the `String?.withoutPrefix()` helper function.

Reply:
I assume you meant the next line right ?